frontend/src/containers/NFTCardContainer.jsx

```diff
@@ -30,16 +31,34 @@ function NFTCardContainer(props) {
     setDetailNFT(detail);
   };
 
+  const alreadyInProcessModal = () => {
+    setModalContent('inProcess');
+    setIsModalShow(true);
+  };
+
   useEffect(() => {
     if (walletAddress !== '') {
       getMyNFTs(walletAddress).then((res) => {
         const nfts = res.data;
         setMyNFTs(nfts);
       });
+
+      getMycollaterals(walletAddress).then((res) => {
+        res.data.forEach((element) => {
+          collaterals.set(element.tokenId, true);
+        });
+      });
```

Comment:
혹시 이렇게 했을 때는 기대 결과와 다르게 동작할까요?
```suggestion
import React, { useEffect, useState } from 'react'
...중략
const collaterals = useRef(new Map)
...중략
getMycollaterals(walletAddress).then((res) => {
  res.data.forEach((element) => {
    collaterals.current.set(element.tokenId, true);
  }
}
...중략
return (
  <NFTCard
    alreadyInProcessModal={alreadyInProcessModal}
    collaterals={collaterals.current}
    data={myNFTs}
    showNFTDetailModal={showNFTDetailModal}
  />
);
```